Services/Services/RecruitmentService.cs

```diff
@@ -4,39 +4,39 @@
 using Data.Repositories;
 using Microsoft.Extensions.Logging;
 using PagedList;
-using Services.DTOs.Recruitment;
+using Data.DTOs.Recruitment;
 using System.Security.Claims;
+using Data.DTOs.RecruitmentSkill;
 
 namespace Services.Services
 {
     public class RecruitmentService
     {
-        private readonly IMapper mapper;
-        private readonly RecruitmentRepository repo;
-        private readonly UserRepository userRepo;
-        private readonly ILogger<RecruitmentService> logger;
-        public RecruitmentService(IMapper map, RecruitmentRepository repo, UserRepository userRepo, ILogger<RecruitmentService> logger)
+        private readonly IMapper _mapper;
+        private readonly RecruitmentRepository _recruitmentRepo;
+        private readonly RecruitmentSkillRepository _recruitmentSkillRepo;
+        private readonly ILogger<RecruitmentService> _logger;
+        public RecruitmentService(IMapper map, RecruitmentRepository repo, RecruitmentSkillRepository recruitmentSkillRepo, ILogger<RecruitmentService> logger)
         {
-            mapper = map;
-            this.repo = repo;
-            this.userRepo = userRepo;
-            this.logger = logger;
+            _mapper = map;
+            this._recruitmentRepo = repo;
+            this._recruitmentSkillRepo = recruitmentSkillRepo;
+            this._logger = logger;
         }
-            
-            
 
-
-        
         public int AddRecruitment(CreateRecruitmentDTO dto)
         {
             try
             {
-                Recruitment recruitment = mapper.Map<Recruitment>(dto);
+                Recruitment recruitment = _mapper.Map<Recruitment>(dto);
```

Comment:
Can you make map for this pair that also will map skills?

Reply:
I wanted to, but I can't make it work, because in the class where I tell AutoMapper how to map one object to the other I would have to map RecruitmentSkillDTO to RecruitmentSkill inside mapping CreateRecruitmentDTO to Recruitment. Either AutoMapper can't do it or I can't make it work